score-client/src/main/java/bio/overture/score/client/ssl/DumbX509TrustManager.java

```diff
@@ -0,0 +1,19 @@
+package bio.overture.score.client.ssl;
+
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import javax.net.ssl.X509TrustManager;
+
+public class DumbX509TrustManager implements X509TrustManager {
+  public DumbX509TrustManager() {}
```

Comment:
Repeating my comment about not calling these `Dumb`. Lets use names that explain the functionality.